spacewalk/config/usr/share/man/man5/rhn.conf.5

```diff
@@ -130,45 +130,6 @@ This parameter controls whether server is allowed to perform system snapshots.
 .B Default:
 1
 
-.TP
-.B "pam_auth_service"
```

Comment:
Just for the record. This configuration parameter still exits but it's pre-populated with a default configuration in the container and users are not supposed to change it.

Reply:
I was contemplating removing rhn conf man page completely as it seems to me it's wonderfully outdated.
Container do not ship it anyway so it's virtually impossible to read it nowadays.

Maybe in different PR.